Assets/Scripts/MoveToClickPoint.cs

```diff
@@ -0,0 +1,56 @@
+// MoveToClickPoint.cs
+using UnityEngine;
+using UnityEngine.AI;
+    
+public class MoveToClickPoint : MonoBehaviour
+{
+    [SerializeField] private NavMeshAgent player;
+    [SerializeField] private Animator anim;
+    [SerializeField] private float distanceToTarget = 2f;
+    
+    private Transform target;
+    private Camera mainCam;
+        
+    void Awake() 
+    {
+        mainCam = Camera.main;
+        player = GetComponent<NavMeshAgent>();
+    }
+        
+    void Update()
+    {
+        anim.SetFloat("isWalking", player.velocity.magnitude/player.speed);
+
+        GetDestination();
```

Comment:
I think this function should return destination since that's what I'd generally expect from a function called GetDestination  😂.

Reply:
I am not sure what you want me to do. Should I make it return a destination somewhere?

Reply:
OK I think i understand what you wanted. I changed the GetDestination to SetTarget .... does that more accurately convey what the method does?

Reply:
It works - in the interest of time, resolving conversation.